portfolio/src/main/webapp/script.js

```diff
@@ -42,9 +42,10 @@ function age() {
 function getMessages() {
```

Comment:
I wonder if maybe this getMessage can be reused for the form to get different pages.